synedrion/src/cggmp21/interactive_signing.rs

```diff
@@ -1009,6 +1090,21 @@ where
             sigma,
         }
     }
+    fn public_aux(&self, i: &I) -> Result<&PublicAuxInfoPrecomputed<P>, LocalError> {
+        self.context
+            .aux_info
+            .public_aux
+            .get(i)
+            .ok_or_else(|| LocalError::new("Missing public_aux for party Id {i:?}"))
+    }
+
+    fn public_shares(&self, i: &I) -> Result<&Point, LocalError> {
+        self.context
```

Comment:
I think you have the same code above where you could just call this method

Reply:
That's on `Round2`, but if I move it to `Context` I could.